packages/components/inputs/selectable-search-input/src/selectable-search-input.spec.js

```diff
@@ -0,0 +1,237 @@
+import { useState } from 'react';
+import SelectableSearchInput from './selectable-search-input';
+import { screen, render, fireEvent } from '../../../../../test/test-utils';
+
+// We use this component to simulate the whole flow of
+// changing a value and formatting on blur.
+const TestComponent = (props) => {
+  const [value, setValue] = useState();
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+  return (
+    <div>
+      <label htmlFor={SelectableSearchInput.getTextInputId('test-id')}>
+        test-label
+      </label>
+      <SelectableSearchInput
+        id={'test-id'}
+        onChange={handleChange}
+        onSubmit={() => {}}
+        value={value}
+        {...props}
+      />
+    </div>
+  );
+};
+
+TestComponent.defaultProps = {
+  id: 'test-id',
+  name: 'test-name',
+  value: {
```

Comment:
🤔 Just an idea but an alternative would be

```js
values: {
    text: 'foo',
    option: 'bar'
}
```

then the structure of options and values feels a bit more symmetrical.

Reply:
Thats a good idea. Here is the [commit](https://github.com/commercetools/ui-kit/pull/2426/commits/340b32043822221ad7ad8efe89933a97f6fe5ce2). Thanks.